### ISSUE TITLE
Add support for some type of error log filtering to make it easier to just sift through errors

### ISSUE THREAD

**Issue by yondonfu** [livepeer/go-livepeer#2104]
On one hand enabling more verbose logging helps with understanding what is happening on the node, on the other hand it makes it really difficult to sift through actual error logs. 

What is a good way to make it easy to *just* access error logs? Maybe have an option to pipe error logs specifically to a file?

**Comment by darkdarkdragon** [comment]
@yondonfu Can you elaborate on that?
What's the context? It is about our infrastructure and logs in Papertrail?
` enabling more verbose logging ` - in terms of `glog` system we're using it's not errors, int's `info`. Output of `glog.Error` is less verbose. So, what part of output do you want to see in log files?

**Comment by yondonfu** [comment]
> Can you elaborate on that?

Yep. A number of orchestrator operators on the network have commented that it is difficult for them to sift through error logs that are sandwiched in between tons of verbose info logs (for example, at least one person recalled seeing a "Generic error in external library" error log, but wasn't sure how to get more information as a part of a report). So, I created this issue to track possible solutions for this problem whether it be a feature on the node that can make this easier for users or a recommendation (maybe docs on how to setup rotating log files and then using grep or something to filter specifically for error logs that can be dumped in a specific file) that we provide to node operators for keeping track of error logs and inspecting them when needed.

**Comment by darkdarkdragon** [comment]
@yondonfu maybe [this](https://github.com/livepeer/go-livepeer/pull/2105) could help. `glog` can log to files, and it also rotates logs file. This PR just exposes existing `glog`'s command line options.
